src/Input/Factory/FieldFactory.php

```diff
@@ -69,6 +70,19 @@ public function create(
 
                     $oneOf[] = $this->create($operationName, $objectName, $this->resolveReference($oneOfSchema), false);
                 }
+            } elseif (isset($schema->anyOf)) {
+                $type = FieldType::SPEC_TYPE_OBJECT;
+                if ($referenceName === '') {
+                    $referenceName = SchemaNaming::getClassName($schemaReference, $fieldName);
+                }
+
+                /** @var Reference $anyOfSchema */
+                foreach ($schema->anyOf as $anyOfSchema) {
```

Comment:
maybe to avoid duplicates we could have just 1 block
if (isset($schema->oneOf) || isset($schema->anyOf)) {
...
then in the foreach ($schema->oneOf ?? $schema->anyOf or smth like that

OR use some private method if you prefer, sending the $schema->oneOf or $schema->anyOf, typehinting with Reference wdyt?